packages/template-retail-react-app/app/pages/account/orders.test.js

```diff
@@ -236,3 +236,172 @@ describe('Handles order with missing or partial data gracefully', () => {
         expect(screen.queryByText(/shipping address/i)).not.toBeInTheDocument()
     })
 })
+
+describe('Order with multiple shipments (pickup and delivery)', () => {
+    let orderNo
+
+    beforeEach(async () => {
+        const mockStore = {
```

Comment:
I am wondering if we can store this data somewhere to reuse? it takes up a bit of space in here